packages/debug/src/browser/debug-session-contribution.ts

```diff
@@ -76,7 +78,19 @@ export class DefaultDebugSessionFactory implements DebugSessionFactory {
     @inject(MessageClient)
     protected readonly messages: MessageClient;
 
+    @inject(OutputChannelManager)
+    protected readonly outputChannelManager: OutputChannelManager;
+
+    @inject(DebugPreferences)
+    protected readonly debugPreferences: DebugPreferences;
+
     get(sessionId: string, options: DebugSessionOptions): DebugSession {
+        let traceOutputChannel: OutputChannel | undefined;
+
+        if (this.debugPreferences['debug.trace']) {
+            traceOutputChannel = this.outputChannelManager.getChannel(`Debug session ${sessionId}`);
```

Comment:
How about using the same channel (`Debug`) for all sessions?

It won't pollute the output view. One can always clear the console to get rid of previous sessions.

Reply:
Ok.  Using the session id as a prefix?  Or a subset of the session id?

Reply:
I don't know what I can do with a session id. It is internal things, as a user you will never see it.

Reply:
Indeed it's not the best.  But for now I think it's enough, it at least lets you differentiate messages from two different connections.  At the moment, I actually need to work on the single-session use case, so it doesn't really matter.